fortran/example/README.md

```diff
@@ -0,0 +1,75 @@
+# How to use spglib example file in Fortran
+
+This is the README file for spglib examples for Fortran.
```

Comment:
Keep the formatting consistent with the previous one.

Reply:
Yes, thanks. Like this d420c5d?

Reply:
:+1: